mousestyles/visualization/plot_clustering.py

```diff
@@ -0,0 +1,129 @@
+from matplotlib import pyplot as plt
```

Comment:
Don't forget your imports since you created a new script.
`from scipy.cluster.hierarchy import dendrogram, linkage`
`import numpy as np`